src/blocks/scratch3_event.js

```diff
@@ -47,7 +47,6 @@ Scratch3EventBlocks.prototype.getHats = function () {
 Scratch3EventBlocks.prototype.hatGreaterThanPredicate = function (args, util) {
     var option = Cast.toString(args.WHENGREATERTHANMENU).toLowerCase();
     var value = Cast.toNumber(args.VALUE);
-    // @todo: Other cases :)
```

Comment:
Same with warnings on TODOs — I mostly made them warnings so that it was easy to see our todos just by linting, not necessarily to suggest that you should remove the comment.